### ISSUE TITLE
Bad version on latest version?

### ISSUE THREAD

**Issue by jmiguelr** [artikell/joplin-excalidraw#11]
Hello!

I was wondering if there was a new release for the plugin supporting some new "original" excalidraw features (curved arrows, for example) so I realized I have installed the 1.3.0 version but in github the latest is 1.4.1. So I manually downloaded it and copied to my `~/.config/joplin-desktop/plugins` directory. Well... nothing happened. So I untar the 1.4.1 `com.joplin.excalidraw.jpl` and I notice the version on manifest is still 1.3.0.  Is that on purpose or just a missing updated file?

**Comment by jmiguelr** [comment]
I've just find https://github.com/artikell/joplin-excalidraw/pull/8 , that would fix it, right?. Looks like it's not a very dangerous PR and it will be great for all plugin users to know the real version we're running.

**Comment by Nschembr** [comment]
This is still not done and joplin is still having issues with the temp folder and the drawing not displaying,  https://github.com/artikell/joplin-excalidraw/issues/2#issuecomment-1464796408

I'm running on linux flatpack and I needed to mkdir -p  the following path,  mkdir -p /run/user/1000/app/net.cozic.joplin_desktop/joplin-excalidraw-plugin/

I'm looking to have the desktop app work the same as android.

Thank you for the cool plugin.

ubuntu mate Ubuntu 22.04.4 LTS
Test procedure:
rm -rf ~/.config/joplin-desktop/
download appimage 
chmod 755 appimage 
./appimage
install joplin-excaldraw
restart
draw line
save
The image fails to show up in notebook preview. 


tested:
./Joplin-2.7.14.AppImage - no plugin
Joplin-2.12.19.AppImage - no preview
Joplin-2.14.22.AppImage - no preview 
Joplin-3.0.8.AppImage    - no preview

I liked this plugin on my phone. 

I have moved to Draw.io 2.2.0 plug-in  until this is fixed.
